packages/conventional-changelog-writer/lib/util.js

```diff
@@ -12,20 +12,20 @@ function compileTemplates (templates) {
   const footerPartial = templates.footerPartial
   const partials = templates.partials
 
-  if (_.isString(headerPartial)) {
+  if (typeof headerPartial === 'string') {
     Handlebars.registerPartial('header', headerPartial)
   }
 
-  if (_.isString(commitPartial)) {
+  if (typeof commitPartial === 'string') {
     Handlebars.registerPartial('commit', commitPartial)
   }
 
-  if (_.isString(footerPartial)) {
+  if (typeof footerPartial === 'string') {
```

Comment:
Same comment as `typeof boolean`:

```
> _.isString(new String('hey'))
true
```

It might be worth making your own helper that checks both `typeof` and `instanceof`.